### PR TITLE
Move rollback section. Fixes #746

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -3222,6 +3181,49 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           below.</t>
         </list></t>
       </section>
+      <section title="Processing a Rollback" anchor="sec.processing-a-rollback">
+        <t>A rollback may be performed if the PeerConnection is in any
+        state except for "stable". This means that both offers and provisional
+        answers can be rolled back. If a rollback is attempted in the "stable"
+        state, processing MUST stop and an error MUST be returned.The effect
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: space before .

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -3222,6 +3181,49 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+        MUST be the same regardless of whether setLocalDescription or
+        setRemoteDescription is called.</t>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: This seems like it is referring to the error discussion in the previous sentence. Suggest moving this to its own paragraph.

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -3222,6 +3181,49 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+        
+        <t>In order to process rollback, a JSEP implementation
+        abandons the current offer/answer transaction, sets the signaling
+        state to "stable", and sets the pending local and/or remote
+        description (see <xref target="sec.pendinglocaldescription" /> and
+        <xref target="sec.pendingremotedescription" />) to null.
+        Any resources or candidates that were allocated by the
+        abandoned local description are discarded; any media that
+        is received will be processed according to the previous
+        local and remote descriptions. Rollback can only be used to
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: This discussion of states should probably move to the para above that discusses valid states.

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -3222,6 +3181,49 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
       </section>
+      <section title="Processing a Rollback" anchor="sec.processing-a-rollback">
+        <t>A rollback may be performed if the PeerConnection is in any
+        state except for "stable". This means that both offers and provisional
+        answers can be rolled back. If a rollback is attempted in the "stable"
+        state, processing MUST stop and an error MUST be returned.The effect
+        MUST be the same regardless of whether setLocalDescription or
+        setRemoteDescription is called.</t>
+        
+        <t>In order to process rollback, a JSEP implementation
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: The future tense used in this section is inconsistent with the surrounding sections. Suggest changing "will be processed" to "is processed" below.

File: draft-ietf-rtcweb-jsep.xml [hunk 5]
```diff
@@ -3222,6 +3181,49 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+        abandoned local description are discarded; any media that
+        is received will be processed according to the previous
+        local and remote descriptions. Rollback can only be used to
+        cancel proposed changes; there is no support for rolling
+        back from a stable state to a previous stable state. Note
+        that this implies that once the answerer has performed
+        setLocalDescription with his answer, this cannot be rolled
+        back.</t>
+
+        <t>A rollback will disassociate any RtpTransceivers that
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: As above, suggest changing "will disassociate" to "disassociates"

**Comment by fluffy** [comment]
LGTM

**Comment by ekr** [comment]
@juberti: PTAL

**Pull request merged**